src/CMakeLists.txt

```diff
@@ -18,6 +18,12 @@ add_executable(bpftrace
 if(HAVE_NAME_TO_HANDLE_AT)
   target_compile_definitions(bpftrace PRIVATE HAVE_NAME_TO_HANDLE_AT=1)
 endif(HAVE_NAME_TO_HANDLE_AT)
+if(HAVE_BCC_PROG_LOAD)
```

Comment:
Any reason to set these definitions here and on `tests/CMakeLists.txt` instead of setting them on the root `CMakeLists.txt`? If we move those definitions to the root `CMakeLists.txt` we can avoid duplicated code.

Reply:
Here I just follow the way how 'HAVE_NAME_TO_HANDLE_AT' is handled.
I prefer to fix exactly one thing in one PR. So what about another PR on this?

Reply:
Sounds good to me